.github/labeler.yml

```diff
@@ -13,6 +13,11 @@
   - '**/animation/*'
   - '**/*animation*'
 
+'a: desktop':
+  - '**/linux/*'
```

Comment:
Should these have an additional directory wildcard at the end?

`  - '**/linux/**/*'`

Reply:
That's what I thought too, but the other paths don't seem to follow that pattern. For example, `material` definitely has nested folders, yet its pattern is [`**/material/*](https://github.com/flutter/flutter/blob/master/.github/labeler.yml#L51). I'll add the additional directory wildcard